models/aliyun_ftasr_api_zh/asr_sdk.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
+# -*- coding: utf8 -*-
+'''
+The code is base on official documentation at:
+    https://help.aliyun.com/document_detail/94062.html
+    录音文件识别 Python Demo
+'''
+import json
+import time
+import sys
+import codecs
+from aliyunsdkcore.acs_exception.exceptions import ClientException
+from aliyunsdkcore.acs_exception.exceptions import ServerException
+from aliyunsdkcore.client import AcsClient
+from aliyunsdkcore.request import CommonRequest
+
+accessKeyId = ""
+with open('accessKeyId', 'r') as f:
+    accessKeyId = f.readline().strip()
+
+accessKeySecret = ""
+with open('accessKeySecret', 'r') as f:
```

Comment:
same kind of renaming as above, refer to https://github.com/SpeechColab/Leaderboard/blob/master/models/aliyun_api_zh/update_token.py#L19